pkg/kotsadmsnapshot/backup.go

```diff
@@ -370,6 +374,7 @@ func CreateInstanceBackup(ctx context.Context, cluster *downstreamtypes.Downstre
 	backupAnnotations["kots.io/kotsadm-deploy-namespace"] = kotsadmNamespace
 	backupAnnotations["kots.io/apps-sequences"] = marshalledAppsSequences
 	backupAnnotations["kots.io/apps-versions"] = marshalledAppVersions
+	backupAnnotations["kots.io/is-airgap"] = strconv.FormatBool(isAirgap)
```

Comment:
how about just using this:
```suggestion
	backupAnnotations["kots.io/is-airgap"] = kotsadm.IsAirgap()
```